VagrantProvisionUbuntu1604.sh

```diff
@@ -212,38 +216,32 @@ EOT
 fi
 
 # gem installs are *very* slow, hence all the checks in place here to facilitate debugging
-gem list --local | grep -q mime-types
-if [ $? -eq 1 ]; then
+if [ `gem list --local | grep -q mime-types` -eq 1 ]; then
    #sudo gem install mime-types -v 2.6.2
    gem install mime-types
 fi
-gem list --local | grep -q cucumber
-if [ $? -eq 1 ]; then
+if [ `gem list --local | grep -q cucumber` -eq 1 ]; then
    #sudo gem install cucumber
    gem install cucumber
 fi
-gem list --local | grep -q capybara-webkit
-if [ $? -eq 1 ]; then
+if [ `gem list --local | grep -q capybara-webkit` -eq 1 ]; then
+   sudo apt-get install qt5-default libqt5webkit5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
```

Comment:
I believe that `capybara-webkit` wants the Qt 4 development headers, not Qt 5 -- the package you're looking for is `libqtwebkit-dev`.